src/osd/scrubber_common.h

```diff
@@ -17,6 +17,9 @@ namespace Scrub {
 /// high/low OP priority
 enum class scrub_prio_t : bool { low_priority = false, high_priority = true };
 
+/// identifying a specific scrub
+using act_token_t = uint32_t;
```

Comment:
This really doesn't clarify anything, I'd rather you simply pass around epoch_t.

Reply:
It does clarify the arguments of the functions involved. I find it clearer, and safer (although for real safety we
could use a real strong type here around an epoch. If you think it would be better, I'd use one of the strong
types libraries. I hope we have already used any in Ceph).

Reply:
Hmm.  I was going to suggest that you use the epoch at which the scrub was reserved, but there's actually no guarrante that an epoch change happens between two scrub attempts from the primary.  A monotonically increasing token here I suppose is as good as we can get then.